### DISCUSSION TITLE
OBS plugin?

### DISCUSSION THREAD

**Discussion by aes44984** [DistroAV/DistroAV#767] [Q&A - Support]
Hello, I am new to computers (relatively). I updated OBS and now cannot connect our NDI camera. I have downloaded the Windows installer but the other two files recommended due to the lack of an app that will run them? I can get the camera to connect via the Video Capture device on OBS but no sound comes through our live stream. I am at a loss.

**Comment by Trouffman** [comment]
Please check the installation guide you are likely missing a per-requisite.
